source/server-selection/server-selection.rst

```diff
@@ -615,7 +643,8 @@ A valid ``$readPreference`` document for mongos has the following requirements:
     - 'secondaryPreferred'
     - 'nearest'
 
-2.  If the ``mode`` field is "primary", the ``tags`` and ``maxStalenessSeconds`` fields MUST be absent.
+2.  If the ``mode`` field is "primary", the ``tags``, ``maxStalenessSeconds``,
+    and ``hedge`` fields MUST be absent.
```

Comment:
For now, I expanded the "Passing ReadPreference to Mongos" prose tests to cover the `hedge` option, as it would be covered by the `$readPreference` handling described there. Since we don't expect server selection to change based on the `hedge` option, I've avoided adding more tests there. Introducing a spec test runner for an option that is essentially passed through and not handled by the driver itself seems overkill.